casper/src/test/scala/coop/rchain/casper/helper/TestNode.scala

```diff
@@ -220,7 +220,7 @@ class TestNode[F[_]](
 
   def shutoff() = transportLayerEff.clear(local)
 
-  def visualizeDag(): F[String] = {
+  def visualizeDag(startBlockNumber: Int): F[String] = {
```

Comment:
What is this number supposed to be? Is it offset relative to the latest block? Or it is `seqNum`?

Reply:
This is just the test. Actually this method is not in use in anywhere. I think it is probably for debugging.